sdl2promela/stop_condition/translator.py

```diff
@@ -15,30 +16,53 @@ class TranslateException(Exception):
     pass
 
 
-@dispatch(model.OrExpression)
-def _generate(expr: model.OrExpression):
-    return (
+class GenerateContext:
+
+    processes: Dict[str, ogAST.Process]
+    choice_selection: str
```

Comment:
there is clear for it, but not set... although there is an external method _set_choice_selection

A description of the function and behaviour of this field would be very welcome.

Reply:
documentation created